packages/@storefront/sayt/src/autocomplete/index.ts

```diff
@@ -128,6 +144,8 @@ class Autocomplete {
   }
 }
 
+type TargetData = { query: string, refinement: string, field: string, pastPurchase: string };
```

Comment:
Should this type be included into the Autocomplete namespace?

Reply:
Good question. Since this type definition is only applied to methods of the `Autocomplete` class itself, which themselves are not particularly relevant in any other context, I would say no. That said, let's see if this is flagged as part of any other review?

Reply:
Fair I will leave this up and see if it flagged by anyone else, but for my part I understand and agree for leaving it as is.

Reply:
CC: @jmbtutor?

Reply:
I think we should, in part because we do for similar cases where we require an extra type. Also, if an implementation were to extend this component, they may want to refer to it as well.

Reply:
Noted. Will update.

CC: @JoeyDeol

Reply:
Updated. Resolving.